[Bug] Docker build for Alpine fails with AssertionError from distutils

**Describe the bug**

The image fails to build due to a Python error, likely related to distutils.

**To Reproduce**

See the last (at the time of writing) commits to the main branch:

* https://github.com/OSGeo/grass/runs/4585030087?check_suite_focus=true
* https://github.com/OSGeo/grass/runs/4598895458?check_suite_focus=true

**Screenshots**

```
#26 [grass 3/6] RUN apk add curl && curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py && python3 get-pip.py pip==20.0.2 && rm get-pip.py
#26 0.286 OK: 343 MiB in 132 packages
#26 0.311   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
#26 0.311                                  Dload  Upload   Total   Spent    Left  Speed
#26 0.311 
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 2108k  100 2108k    0     0  14.4M      0 --:--:-- --:--:-- --:--:-- 14.5M
#26 3.140 /usr/lib/python3.8/site-packages/_distutils_hack/__init__.py:23: UserWarning: Distutils was imported before Setuptools, but importing Setuptools also replaces the `distutils` module in `sys.modules`. This may lead to undesirable behaviors or errors. To avoid these issues, avoid using distutils directly, ensure that setuptools is installed in the traditional way (e.g. not an editable install), and/or make sure that setuptools is always imported before distutils.
#26 3.140   warnings.warn(
#26 3.140 /usr/lib/python3.8/site-packages/_distutils_hack/__init__.py:35: UserWarning: Setuptools is replacing distutils.
#26 3.140   warnings.warn("Setuptools is replacing distutils.")
#26 3.151 Traceback (most recent call last):
#26 3.151   File "get-pip.py", line 27081, in <module>
#26 3.151     main()
#26 3.151   File "get-pip.py", line 139, in main
#26 3.151     bootstrap(tmpdir=tmpdir)
#26 3.151   File "get-pip.py", line 120, in bootstrap
#26 3.151     args = determine_pip_install_arguments()
#26 3.151   File "get-pip.py", line 65, in determine_pip_install_arguments
#26 3.151     import setuptools  # noqa
#26 3.151   File "/usr/lib/python3.8/site-packages/setuptools/__init__.py", line 8, in <module>
#26 3.151     import _distutils_hack.override  # noqa: F401
#26 3.151   File "/usr/lib/python3.8/site-packages/_distutils_hack/override.py", line 1, in <module>
#26 3.151     __import__('_distutils_hack').do_override()
#26 3.151   File "/usr/lib/python3.8/site-packages/_distutils_hack/__init__.py", line 73, in do_override
#26 3.151     ensure_local_distutils()
#26 3.151   File "/usr/lib/python3.8/site-packages/_distutils_hack/__init__.py", line 61, in ensure_local_distutils
#26 3.151     assert '_distutils' in core.__file__, core.__file__
#26 3.151 AssertionError: /usr/lib/python3.8/distutils/core.py
#26 ERROR: process "/bin/sh -c apk add curl && curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py && python3 get-pip.py pip==20.0.2 && rm get-pip.py" did not complete successfully: exit code: 1
------
 > [grass 3/6] RUN apk add curl && curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py && python3 get-pip.py pip==20.0.2 && rm get-pip.py:
#26 3.151     import setuptools  # noqa
#26 3.151   File "/usr/lib/python3.8/site-packages/setuptools/__init__.py", line 8, in <module>
#26 3.151     import _distutils_hack.override  # noqa: F401
#26 3.151   File "/usr/lib/python3.8/site-packages/_distutils_hack/override.py", line 1, in <module>
#26 3.151     __import__('_distutils_hack').do_override()
#26 3.151   File "/usr/lib/python3.8/site-packages/_distutils_hack/__init__.py", line 73, in do_override
#26 3.151     ensure_local_distutils()
#26 3.151   File "/usr/lib/python3.8/site-packages/_distutils_hack/__init__.py", line 61, in ensure_local_distutils
#26 3.151     assert '_distutils' in core.__file__, core.__file__
#26 3.151 AssertionError: /usr/lib/python3.8/distutils/core.py
------
Dockerfile:189
--------------------
 187 |     COPY --from=build /usr/local/grass* /usr/local/grass/
 188 |     # pip 20.0.0 fix
 189 | >>> RUN apk add curl && curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py && python3 get-pip.py pip==20.0.2 && rm get-pip.py
 190 |     
 191 |     # install external Python API
--------------------
error: failed to solve: process "/bin/sh -c apk add curl && curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py && python3 get-pip.py pip==20.0.2 && rm get-pip.py" did not complete successfully: exit code: 1
Error: buildx failed with: error: failed to solve: process "/bin/sh -c apk add curl && curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py && python3 get-pip.py pip==20.0.2 && rm get-pip.py" did not complete successfully: exit code: 1
```

**System description**

The main branch, alpine image.

**Additional context**

CI for pushes/merges/commits to the main branch is now failing.

--- a/docker/alpine/Dockerfile
+++ b/docker/alpine/Dockerfile
@@ -185,8 +185,6 @@ ENV LC_ALL="en_US.UTF-8"
 # Copy GRASS GIS from build image
 COPY --from=build /usr/local/bin/grass /usr/local/bin/grass
 COPY --from=build /usr/local/grass* /usr/local/grass/
-# pip 20.0.0 fix
-RUN apk add curl && curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py && python3 get-pip.py pip==20.0.2 && rm get-pip.py
 
 # install external Python API
 RUN pip3 install --upgrade pip six grass-session --ignore-installed six

